src/modules/account/RadixAccount.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai'
+import 'mocha'
+import { RadixAccount } from '../..'
+
+describe('RadixAccount', () => {
```

Comment:
If you're also doing this in java, would be good to test one seed and output pair from there, to check that we get the same result in both libraries.